code/go/0chain.net/smartcontract/zcnsc/sc.go

```diff
@@ -0,0 +1,81 @@
+package zcnsc
+
+import (
+	"context"
+	"net/url"
+
+	cstate "0chain.net/chaincore/chain/state"
+	"0chain.net/chaincore/smartcontract"
+	"0chain.net/chaincore/smartcontractinterface"
+	"0chain.net/chaincore/transaction"
+	"0chain.net/core/common"
+	// metrics "github.com/rcrowley/go-metrics"
+)
+
+const (
+	// ADDRESS ...
+	ADDRESS = "6dba10422e368813802877a85039d3985d96760ed844092319743fb3a76712e0"
+	name    = "zcn"
+)
+
+// ZCNSmartContract ...
+type ZCNSmartContract struct {
+	*smartcontractinterface.SmartContract
+}
+
+func NewZCNSmartContract() smartcontractinterface.SmartContractInterface {
+	var zcnCopy = &ZCNSmartContract{
+		smartcontractinterface.NewSC(ADDRESS),
+	}
+	zcnCopy.setSC(zcnCopy.SmartContract, &smartcontract.BCContext{})
+	return zcnCopy
+}
+
+// InitSC ...
+func (zcn *ZCNSmartContract) InitSC() {}
+
+// SetSC ...
+func (zcn *ZCNSmartContract) setSC(sc *smartcontractinterface.SmartContract, bcContext smartcontractinterface.BCContextI) {
+	zcn.SmartContract = sc
+	zcn.SmartContract.RestHandlers["/globalPerodicLimit"] = zcn.globalPerodicLimit
```

Comment:
Typo `Periodic`. What is periodic about the cofig values returned?

Reply:
Makes sense. Removed as useless